ppr-ui/src/components/search/SearchBar.vue

```diff
@@ -393,11 +393,22 @@ export default defineComponent({
       })
     })
 
+    const cleanUpInput = (dirtyValue: string|undefined) => {
+      if (dirtyValue === undefined) {
+        return undefined
+      }
+      return dirtyValue
+        .trim()
+        .replaceAll(/[\u200B-\u200D\uFEFF\u200E\u200F]|(?:&#x200E;)/g, '')
+        .replaceAll(/[\u2018\u2019]/g, "'")
+        .replaceAll(/[\u201C\u201D]/g, '"')
+    }
+
     const getCriteria = () => {
       if (localState.isIndividual) {
-        const first = localState.searchValueFirst?.trim()
-        const second = localState.searchValueSecond?.trim()
-        const last = localState.searchValueLast?.trim()
+        const first = cleanUpInput(localState.searchValueFirst)
+        const second = cleanUpInput(localState.searchValueSecond)
+        const last = cleanUpInput(localState.searchValueLast)
```

Comment:
If the function returns `undefined`for any of these properties as it could, do we still want to return the search criteria with `undefined` properties?

Reply:
I believe that is the original behavior.
If it works don't touch it?
I'm only making sure `cleanUpInput` is behaving just like its original expression, just with a bit more of string manipulation.

Reply:
haha fair enough. You are right! (As long as it worked to begin with 😆 )
Resolving.